src/Integration/Suppression/ChecksumCalculator.cs

```diff
@@ -41,7 +40,7 @@ public static string Calculate(string text)
             }
 
             string content = Regex.Replace(text, "\\s", ""); // strip whitespace
-            using (MD5 md5Digest = MD5.Create())
+            using (var md5Digest = new Md5Managed())
```

Comment:
As previously discussed, what validation do you think we need to do to satisfy ourselves that the algorithm is is correctly implemented?

Reply:
We are using it for suppression matching so I think we should focus on that. I tried C++/C# suppressions, long lines, short lines, variables, comments. I'll try unicode characters. Anything else do you think we should test?

Reply:
Those sound like the right sort of things to test. I was expecting to see some unit tests comparing the results against the .NET implementation.
Did you consider using an existing NuGet package like https://www.nuget.org/packages/BouncyCastle/ or https://www.nuget.org/packages/CryptSharpOfficial/ that have seem to be popular?